_inc/jetpack-modules.js

```diff
@@ -59,6 +59,15 @@
 		$( '.modal' ).focus();
 	};
 
+	/**
+	 * If modalinfo is defined, auto popup the modal
+	 */
+	$( window ).load(function() {
```

Comment:
@zinigor yes, it doesn't work with `$( document ).ready`. It has to do with the modal not being loaded yet at that point